editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -684,10 +684,8 @@ function useSelectOrLiveModeSelectAndHover(
         const foundTargetIsSelected = foundTarget?.isSelected ?? false
 
         if (foundTarget != null && foundTargetIsSelected && doubleClick) {
-          // for components without passed children doubleclicking enters focus mode
-          const isFocusableLeaf = MetadataUtils.isManuallyFocusableLeafComponent(
+          const isFocusableLeaf = MetadataUtils.isManuallyFocusableComponent(
```

Comment:
minor - maybe change the name since it's not only leaves now

Reply:
Good shout! Done